pset-functions.js

```diff
@@ -136,6 +194,45 @@ console.log('Problem 3:')
 
 // Add your code below this line
 
+const scrabbleWord = "javascript"
+
+function scrabbleScore(scrabbleWord) {
+  const scrabbleLetters = scrabbleWord.split('')
+
+  let letterPoints = scrabbleLetters.map(letter => {
```

Comment:
🛠 Read the comment below and update your code accordingly

@Defy1x the logic looks great, however, the code is hard to scan due to the indentation issues

please address the indentation issues for this problem

Also, best practice for structuring `if...else if` is the following:

```
if (condition) {

} else if (condition) { //<- as many of these as are needed

}  else { 

} 
```
update your code to adhere to this approach

Reply:
Done